CODE_OF_CONDUCT.md

```diff
@@ -69,7 +69,7 @@ Read carefully our Contributing Guidelines to know how to contribute properly in
 project. Members and maintainers must adhere to some rules regarding to pull requests
 reviews and creation of issues and pull requests:
 
-* During code reviews do not comment on coding standards and styles -focus on algorithmical,
+* During code reviews do not comment on coding standards and styles -focus on algorithmically,
```

Comment:
I think correct version should be:
```
During code reviews do not comment on coding standards and styles -focus on algorithmic,
structural or naming issues-, help to solve problem.
```

Reply:
> I think correct version should be:
> 
> ```
> During code reviews do not comment on coding standards and styles -focus on algorithmic,
> structural or naming issues-, help to solve problem.
> ```

yes, you are correct

Reply:
@romashka-btc please update this line with `algorithmic` instead of `algorithmically`